v03_pipeline/deploy/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM docker:dind as BUILD
```

Comment:
This is the actual new Dockerfile, replacing https://github.com/broadinstitute/seqr-loading-pipelines/blob/main/docker/Dockerfile